playbook/run_consumer.go

```diff
@@ -0,0 +1,50 @@
+package playbook
+
+import (
+	"fmt"
+
+	"github.com/flanksource/duty/models"
+	"github.com/flanksource/incident-commander/api"
+	"github.com/flanksource/incident-commander/events"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"gorm.io/gorm"
+)
+
+func NewEventQueueConsumer(db *gorm.DB, pool *pgxpool.Pool) *events.EventConsumer {
+	return events.NewEventConsumer(db, pool, "playbook_run_updates", EventConsumer).WithNumConsumers(5)
+}
+
+func EventConsumer(ctx *api.Context, batchSize int) error {
+	tx := ctx.DB().Begin()
+	if tx.Error != nil {
+		return fmt.Errorf("error initiating db tx: %w", tx.Error)
+	}
+	defer tx.Rollback()
```

Comment:
Is this safe to run after doing a `tx.Commit()` ? does it fail silently in that case ?

Reply:
yea. Once it's committed, the rollback is a no-op.